src/main/java/io/strimzi/kafka/metrics/HttpServers.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.metrics;
+
+import io.prometheus.metrics.exporter.httpserver.HTTPServer;
+import io.prometheus.metrics.model.registry.PrometheusRegistry;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Static class to keep track of all the HTTP servers started by all the Kafka components in a JVM.
```

Comment:
What is static on it?

Reply:
Right, "static" is not the right term. I meant it as a utility. But I guess we can just say "Class to keep track ..."